Framework/Crystal/test/AnvredCorrectionTest.h

```diff
@@ -138,6 +138,24 @@ class AnvredCorrectionTestPerformance : public CxxTest::TestSuite {
 
   void testNoEventsPerformace() { do_test_events(workspace, false, performance); }
 
+  void test_extrapolation_large_muR() {
```

Comment:
Looks like this is currently in the performance test suite. I think it should be in the main AnvredCorrectionTest class?

Reply:
Indeed, don't know how I ended up putting it there sorry!